vendor_import.py

```
@@ -0,0 +1,248 @@
+        create_sql = f'CREATE TABLE IF NOT EXISTS "{table_name}" ({col_defs});'
+        with conn.cursor() as cur:
+            cur.execute(create_sql)
+            conn.commit()
+        print(f"[INFO] Created table: {table_name}")
+        # Insert data
+        for _, row in df.iterrows():
+            values = [
+                str(row[col]) if pd.notnull(row[col]) else None for col in columns
+            ]
+            placeholders = ", ".join(["%s"] * len(columns))
+            insert_sql = f'INSERT INTO "{table_name}" ({', '.join([f'"{col}"' for col in columns])}) VALUES ({placeholders});'
```

Pylint (3.3.7)

[error] Parsing failed: 'f-string: expecting '}' (vendor_import, line 237)'

(E0001)